lib/actions/grants/refresh_token.js

```diff
@@ -99,7 +109,9 @@ module.exports.handler = function getRefreshTokenHandler(provider) {
         refreshTokenValue = await refreshToken.save();
         ctx.oidc.entity('RefreshToken', refreshToken);
       } catch (err) {
-        await refreshToken.destroy();
+        if (conf.refreshTokenHardDelete) {
```

Comment:
Curious, is there a specific reason/use case that we making this configurable?

Reply:
in case this blows up and we need to start cleaning up fast.